openml-lightgbm/lightgbm-provider/src/main/java/com/feedzai/openml/provider/lightgbm/LightGBMBinaryClassificationModel.java

```diff
@@ -102,31 +102,42 @@ public DatasetSchema getSchema() {
     public void close() throws Exception {
     }
 
+    /**
+     * Get features contributions double [ ].
+     *
+     * @param instance the instance
+     * @return the double [ ]
+     * @since 1.2.2
+     */
+    public double[] getFeatureContributions(final Instance instance) {
```

Comment:
to discuss in private, to understand how will we call this method from feedzai products